game.sy

```diff
@@ -252,28 +241,52 @@ init :: fn {
     }
 }
 
+num_players := 0
+player_join :: fn {
+    // TODO(ed): This is a bad idea - since we can
+    // have a race condition here.
+    // We probably want to respond to ONLY that client.
```

Comment:
+1 for this idea. Let's add this functionality if we find another place where responding to a single client directly is necessary. We already have a nice potential ID -- the index in the client handle vec.